question/bank/bulkmove/tests/behat/bulk_move.feature

```diff
@@ -0,0 +1,34 @@
+@qbank @qbank_bulkmove
```

Comment:
Will need a behat test to cover actually moving something in bulk

Reply:
covered by using the api